docs/pages/web/dropdown/show_case.js

```diff
@@ -0,0 +1,536 @@
+            defaultCode={`
+function ScrollableContainerB() {
+  const [isOpen, setIsOpen] = React.useState(false);
+  const [selectedElement, setSelectedElement] = React.useState(null);
+  const [elements, setElements] = React.useState([]);
+  const anchorRef = React.useRef(null);
+
+  React.useEffect(() => {
+    setElements(new Array(20).fill(undefined));
+  }, [setElements]);
+
+  const handleSelect = ({ item }) => {
+    setSelectedElement(item);
+  };
+
+  const preRenderItems = () =>
+    elements.map((item, index) => {
+      const text = \`item \${index}\`;
+      return (
+        <Dropdown.Item
+          key={text}
+          onSelect={handleSelect}
+          option={{ value: text, label: text }}
+          selected={selectedElement}
+        />
+      );
+    });
+
+  return (
+    <Box padding={8} height={500} overflow="scrollY" color="successBase" width="100%">
+      <Box height={300} color="errorBase" overflow="hidden">
+        <ScrollBoundaryContainer>
+          <Button
+            text="Menu open button"
+            onClick={() => setIsOpen(!isOpen)}
+            ref={anchorRef}
+            selected={isOpen}
+            size="lg"
+          />
+          {isOpen && (
+            <Dropdown
+              anchor={anchorRef.current}
+              id="demo-dropdown-example"
+              onDismiss={() => setIsOpen(!isOpen)}
+              idealDirection="up"
+            >
+              {preRenderItems()}
+            </Dropdown>
+          )}
+        </ScrollBoundaryContainer>
+      </Box>
+    </Box>
+  );
+}
+            `}
+          />
+          <MainSection.Card
+            cardSize="lg"
+            title="Page Header"
+            description="Dropdown inner Page Header"
+            defaultCode={`
+function PageHeaderExample() {
+  const [selectedElement, setSelectedElement] = React.useState(null);
+  const [elements, setElements] = React.useState([]);
+
+  React.useEffect(() => {
+    setElements(new Array(10).fill(undefined));
+  }, [setElements]);
+
+  const handleSelect = ({ item }) => {
+    setSelectedElement(item);
+  };
+
+  const preRenderItems = () =>
+    elements.map((item, index) => {
+      const text = \`item \${index}\`;
+      return (
+        <Dropdown.Item
+          key={text}
+          onSelect={handleSelect}
+          option={{ value: text, label: text }}
+          selected={selectedElement}
+        />
+      );
+    });
+
+  return (
+    <Box height={500} color="lightWash" width="100%">
+      <PageHeader
+        title="Dropdown Showcase"
+        primaryAction={{
+          component: <Button text="Menu open button" size="lg" />,
+          dropdownItems: preRenderItems(),
+        }}
+        dropdownAccessibilityLabel="More options"
+      />
+    </Box>
+  );
+}
+            `}
+          />
+          <MainSection.Card
+            cardSize="lg"
+            title="Fixed button"
```

Comment:
Where are these buttons?

Reply:
How we fixed the position of container and the current playground use the entire page to render the component, when we FIXED a position this component going to page edges. =/ 
Example removed, keeping focus on BUG's